listenbrainz/webserver/static/js/src/brainzplayer/BrainzPlayerUI.tsx

```diff
@@ -189,6 +190,13 @@ const BrainzPlayerUI = (
             </span>
           )}
         </div>
+        {isPlayingATrack && (
+          <div className="elapsed small text-muted">
+            {`${isNumber(progressMs) && msToTime(progressMs)}\u202f/\u202f${
+              isNumber(durationMs) && msToTime(durationMs)
+            }`}
```

Comment:
Hmm.. I see how it can render `false / false`. But like you mentioned, there is no way for it to render anything else. Therefore, I'll go lazy and get rid of the check altogether.

I liked the idea of returning "-" string though, but again, I don't see a situation where it's needed to do that check in the `msToTime` util. I trust myself enough that I'll break the perfectly working piece of code in the process 😛 

I used the unicode because using HTML entity in JSX can sometimes cause unexpected behavior. Also, `&nnbsp;` doesn't seem to be working. I tried it on a couple browsers on my machine. it just renders the literal `&nnbsp;`. I also couldn't find it on [fileformat.info](https://www.fileformat.info/info/unicode/char/202f/index.htm) or on [unicode-table.com](https://unicode-table.com/en/202F/). `&thinsp;` works, but it isn't non-breaking. So I decided to go with the hex version of NNBSP which is `&#8239;`, which is also mentioned on unicode-table.com.